src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/sparkrecords/TransientFieldPhysicalLocation.java

```diff
@@ -37,13 +37,13 @@ public TransientFieldPhysicalLocation(int partitionIndex, String name) {
     public int getX() { return this.x; }
 
     @Override
-    public void setX(final int x) { this.x = x; }
```

Comment:
I would add a comment saying that in practice picard always uses short here even though the api is int.

Reply:
done